nidx/nidx_relation/tests/common.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2021 Bosutech XXI S.L.
```

Comment:
Same as the other PRs, `cargo test` will try to run tests in this file. Use common/mod.rs to avoid it